src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy

```diff
@@ -22,8 +22,12 @@ class ComposeDown extends DefaultTask {
             }
             if (extension.removeContainers) {
                 if (getDockerComposeVersion() >= VersionNumber.parse('1.6.0')) {
+                    String[] args = ['down']
+                    if(extension.removeImages) {
+                        args += ['--rmi', 'all', '--volumes']

```

Comment:
Please add `--volumes` options as default (so have `String[] args = ['down', '--volumes']`). Or add also property for volumes removal. Thanks!